cpp/src/pypowsybl.h

```diff
@@ -132,9 +132,10 @@ enum ConnectedComponentMode {
     ALL,
 };
 
-enum XnecSelectionStrategy {
-    ONLY_INTERCONNECTIONS = 0,
-    INTERCONNECTION_OR_ZONE_TO_ZONE_PTDF_GT_5PC,
+enum DefaultXnecProvider {
+    GT_5_PERC_ZONE_TO_ZONE_PTDF = 0,
+    ALL_BRANCHES,
+    INTERCONNECTIONS,
```

Comment:
Sure about that ? is it a breaking change ?

Reply:
Yes, I forgot to remove those in #536. This PR contains breaking changes